### PR TITLE
[WIP] Adding svelte client side

### PR DESCRIPTION
**Pull request by cbarber**
I'm adding svelte along side the react components and splitting the bundle with a `/svelte` route. Components won't be a perfect 1:1 since I'm also rethinking some of the DOM and css which I'll backport to react.

### CODE REVIEW

File: app/javascript/components/side-panel.svelte [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  export let isPaused;
+
+  $: enabledWidgets = widgets.enabled;
+
+  import Date from '@components/date.svelte';
+  import Time from '@components/time.svelte';
+  import filterFragments from '@lib/filter-fragments';
+</script>
+
+<style>
```
Comment (geraldalewis) [inline app/javascript/components/side-panel.svelte]: I know it bucks the `svelte` convention, but have you tried putting `style` code at the bottom under the markup (`react-native` style)? I like it, but YMMV

File: app/javascript/components/time.svelte [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  let time = new Date();
+  $: [hourMin, amPm] = timeForTimezone(timezone, time).split(' ');
+
+  onMount(() => {
+    const interval = setInterval(() => {
+      time = new Date();
+    }, 1000);
+
+    return () => {
+      clearInterval(interval);
```
Comment (geraldalewis) [inline app/javascript/components/time.svelte]: Love `svelte`'s reasonable lifecycle methods and collocated cleanup code 👍 Have you noticed if this clears up the memory leak issue?

File: app/javascript/components/side-panel.svelte (hunk 1 above)
Comment (bencates) [reply app/javascript/components/side-panel.svelte:46]: +1. Prettier has changed their default in the last few months from script-style-template to docblock-script-template-style.

**Comment by cbarber** [comment]
RIP svelte conversion

**Pull request closed without merging**